source/core/parameters.cc

```diff
@@ -566,6 +572,7 @@ namespace Parameters
       grid_arguments = prm.get("grid arguments");
 
       refine_until_target_size = prm.get_bool("enable target size");
+      simplex                  = prm.get_bool("enable simplex");
```

Comment:
I find the "enable simplex" name a bit redundant. I would just put simplex as the parameter and not "enable simplex".

Reply:
It is done